### ISSUE TITLE
Multiple animations

### ISSUE THREAD

**Issue by chuckhacker** [godotengine/collada-exporter#48]
Seems like this only supports one animation

**Comment by set-killer** [comment]
No.
Use the Dope Sheet editor of Blender to create your multiple animations per model. 
After that you can export all animations which you have.

**Comment by chuckhacker** [comment]
No, have to use actions instead of f-curves for any animations to get exported in the XML. Dope Sheet is just UI, not sure how it is relevant.

**Comment by set-killer** [comment]
Dope sheet supports action animations... Almost the same as NLA editor...

**Comment by set-killer** [comment]
Basically when you open the dope sheet editor, then from the 'dope sheet' drop down select 'Action Editor'. Then to the right of the 'Action editor' a 'New' button will appear. Clicking it will create new action animation where you can set the name of the animation like 'walking', 'jumping'... After you may create more animations with the '+' button.

![untitled](https://user-images.githubusercontent.com/4090555/27289501-d1192ba8-5512-11e7-9e36-8f5df393658a.png)

**Comment by chuckhacker** [comment]
Right, and the issue is that if we use f-curves, nothing gets exported.

**Comment by Dvdhyh** [comment]
I am almost having the same issue. I have idle, walk and jump animations. Only the currently select f curve gets exported

**Comment by markoi** [comment]
Maybe I am wrong ... but it seems to me that nothing is exported from the blender action editor if no bones are used for the animation. 

For simple scale/move/rotation there is only (ONE) default animation exported

From the exporter code:

`if (self.config["use_anim_action_all"] and len(self.skeletons)):`

[https://github.com/godotengine/collada-exporter/blob/836241ad09a96b4e12226c4f1592d25d997ad118/io_scene_dae/export_dae.py#L1769](url)

then if len(self.skeletons) is zero... no animation clips are exported/created

But multiple animations without necessarily using bones should be also possible to export...  ?

**Comment by spongeboburu** [comment]
Godot supports animation for non-bones so it should.

It's just like you say, the code looks as if it has only been designed to export all actions for skeletons.

I'm going to see if there is an easy fix, but it looks like that part of the code relies on there being bones quite heavily.

**Comment by minifigmaster125** [comment]
+1 if this can happen, hope it's not too hard to export multiple actions that are not skeletally based.

**Comment by rvenson** [comment]
Is there any solution to this problem? I was able to export all the animations once, but I do not even know what I did, because now only one animation (usually the one selected in the NLA) can be imported
